SETUP/tests/unittests/phpunit_bootstrap.php

```diff
@@ -8,6 +8,8 @@
 
 // Reconnect to the test DB
 DPDatabase::close();
+// Note: The connect arguments need to be kept in sync with DPDatabaseTest.php
```

Comment:
This is true but it's true also for `smoketests/test_tables.sql` and `ci_configuration.sh` -- we should either call out all the places or none of them so we don't mislead readers. Could just say:
```
// Note: These values need to be kept in sync with other files in SETUP/tests/
```

Reply:
Sure, updated the comment to your suggestion.